inst/htmljs/animint.js

```diff
@@ -6,10 +6,11 @@
 // Constructor for animint Object.
 var animint = function (to_select, json_file) {
 
-  function wait_until_then(timeout, condFun, readyFun) {
+   function wait_until_then(timeout, condFun, readyFun) {
```

Comment:
This change is to support functions that need to be run asynchronously and take arguments